### ISSUE TITLE
Issues when replicate the annotation pipeline

### ISSUE THREAD

**Issue by MOMOYATW** [DNA-Rendering/DNA-Rendering#11]
Hello, I am currently working with a pipeline based on xrmocap for DNA-rendering and I've encountered some issues when replicate the annotation pipeline in your paper.

1. Running the pipeline requires setting numerous parameters, could you share the configuration file used in the paper?  For instance, strategies for optimizing 3D keypoints and weights for shape joint loss in SMPL.
2. The paper mentions several keypoints optimization methods, such as 1) keypoint selection, 2) bone length constraint, and 3) three additional filters: a 3D bounding box filter, a movement filter, and a relative position filter. The first two methods are reflected in xrmocap, but the third part is not. Could you share the code or a detailed description of how it's implemented?
3. While using xrmocap for visualizing 2D keypoints and 3D keypoints, I found that the keypoints displayed are noticeably different from those in the dataset, particularly on the feet. The whole-body model uses three points per foot, whereas the dataset shows more points (five), and the number of points on the body also differs. Could you please explain this?

**Comment by DNA-Rendering** [comment]
Hi @MOMOYATW ,

Thanks for your interest in DNA-Rendering! 

1. In fact, there is no common configuration for all cases. We adjusted parameters for each case during the optimization of the SMPLx parameters to achieve the best results.

2. The third method is also presented in xrmocap. For instance, the constraints of the bounding box can be found at https://github.com/openxrlab/xrmocap/blob/3a6b40397f9e3d36c87c11c6c8a3e435c7b0a094/xrmocap/transform/keypoints3d/optim/prior_optimizer.py#L171-L175,  and the process of interpolation can be viewed at https://github.com/openxrlab/xrmocap/blob/main/xrmocap/transform/keypoints3d/optim/nan_interpolation.py

3. In our final implementation, we adopted the same foot representation as coco-wholebody (3 joints for each foot).

Hope it helps!
